### PR TITLE
feat: Allow to mute loggers

### PR DESCRIPTION
**Pull request by marikaner**
## Context

I am quite annoyed by the many log outputs we get in the tests. This is the first step towards making them disappear.

## Definition of Done

Please consider all items and remove only if not applicable.

- [x] Tests created/adjusted for your changes.
- [x] Release notes updated.
  * Provide sufficient context so that each entry can be understood on its own.
  * Be specific about names of functions, classes, modules, etc.
  * Describe when or where this is relevant
  * Use indicative and present tense. For example, write "Provide function `name` that does X in order to Y" over "Now X can be done by calling a new function".
- [ ] PR title adheres to [conventional commit guidelines](https://www.conventionalcommits.org) (please note that only `fix:` and `feat:` will end up in the release notes)
- [ ] If applicable: Properly documented (JSDoc of public API)
- [ ] If applicable: Check if `node run doc` still works.

### CODE REVIEW

File: packages/util/src/logger/cloud-sdk-logger.ts [hunk 1]
```diff
@@ -16,12 +16,36 @@ const container = new Container();
...
 const customLogLevels = {};
 const DEFAULT_LOGGER__MESSAGE_CONTEXT = '__DEFAULT_LOGGER__MESSAGE_CONTEXT';
+let silent = false;
 
 const moduleLogger = createLogger({
   package: 'util',
   messageContext: 'cloud-sdk-logger'
 });
 
+function toggleMuteLoggers(silence: boolean) {
```
Comment (FrankEssenberger) [inline packages/util/src/logger/cloud-sdk-logger.ts:26]: Minor: Method is not exported so not so important, but I am a bit confused `silent` and `silence` both a boolean muting so why not use the same name.

Comment (marikaner) [reply packages/util/src/logger/cloud-sdk-logger.ts:26]: Because silent is a local variable 😕 I just didn't have a better idea.

Comment (FrankEssenberger) [reply packages/util/src/logger/cloud-sdk-logger.ts:26]: ah shadow, consider using a namespace for the private scope above.

**Review by FrankEssenberger — APPROVED** [review]
LGTM

**Comment by marikaner** [comment]
I checked how I would do it with a namespace and it is not as straight forward as I would think. I guess variables cannot be changed from the outside (except using a method). I will refrain from changing to namespaces right now, but if it is important to you, lets talk about another PR.

```
namespace CloudSDKLogger {
  export const format = process.env.VCAP_SERVICES ? kibana : local;
  export const loggerReference = 'sap-cloud-sdk-logger';
  export const exceptionLoggerId = 'sap-cloud-sdk-exception-logger';

  export const container = new Container();
  export const exceptionTransport = new transports.Console();
  export const customLogLevels = {};
  export const DEFAULT_LOGGER__MESSAGE_CONTEXT =
    '__DEFAULT_LOGGER__MESSAGE_CONTEXT';
  export let silent = false;
}
```

**Pull request merged**